.github/workflows/ci.yml

```
@@ -119,6 +119,7 @@ jobs:
   success:
     runs-on: ubuntu-latest
     needs: run-systems
+    if: always()
```

If we want it to not run when the job was cancelled, we might want ...

```suggestion
    if: success() || failure()
```

---

I don't want to think about how that would interact; it's safer to just say "always run this", since we'll only exit with a failure if the following `if` condition is satisfied.